src/cli.rs

```diff
@@ -497,6 +516,31 @@ pub struct Opt {
     #[structopt(long = "line-numbers-right-style", default_value = "auto")]
     pub line_numbers_right_style: String,
 
+    /// Maximum number of wrapped lines to display. Any content that still does not fit will be truncated.
+    /// A value of 0 means no limit. Only used in side-by-side mode.
+    #[structopt(long = "wrap-max-lines", default_value = "3")]
+    pub wrap_max_lines: usize,
```

Comment:
I still have a slight concern that some users will experience a mental off-by-one here. I'm not expressing a hard opinion here, just opening discussion. My brain at least partly expects

- `wrap-max-lines=1`: one normal line and one "wrapped" line, so 2 lines total
- `wrap-max-lines=0`: just one normal line, no wrapped lines. In other words: disable wrapping
- `wrap-max-lines=-1`: there is no maximum to the number of wrapped lines